vars/checksPublishResults.groovy

```diff
@@ -12,11 +12,6 @@ import groovy.transform.Field
 @Field def STEP_NAME = getClass().getName()
 
 @Field Set TOOLS = [
-    /**
-     * Allows to publish the check results.
-     * @possibleValues `true`, `false`, `Map`
-     */
-    'aggregation',
```

Comment:
Should the step fail if this key is still defined?

Reply:
Good idea as it was also possible to define thresholds here.